.gitignore

```diff
@@ -1 +1,129 @@
-TODO
```

Comment:
Just remove all this stuff then and leave it as TODO? Or am I missing something?